stages/org.osbuild.bootupd

```diff
@@ -99,22 +115,11 @@ def main(args, options):
     # want to make sure the version used matches the target and not
     # risk any inconsistencies with the build root). Let's set up
     # and chroot to run the bootupctl command from the target.
-    submounts = ['dev', 'proc', 'sys', 'run', 'var', 'tmp']
-    for mnt in submounts:
-        subprocess.run(['mount', '--rbind',
-                       os.path.join("/", mnt),
-                       os.path.join(root, mnt)],
-                       check=True)
-    try:
+    with bind_mounts(['/dev', '/proc', '/sys', '/run', '/var', '/tmp'], root):
```

Comment:
```suggestion
    with bind_mounts(['dev', 'proc', 'sys', 'run', 'var', 'tmp'], root):
```

Feels a bit weird to have the `/` in the specified mounts here but then just `lstrip("/") them off for each use.. Is that just the desire to be as explicit as possible that they are a toplevel mount?

Reply:
It's fine as is, I just wanted to understand more.